index.js

```diff
@@ -78,6 +81,7 @@ const build = (tagName, attrs, children) => {
 };
 
 function h(tagName, attrs) {
+	// eslint-disable-next-line prefer-rest-params
```

Comment:
Why are you disabling this?

Reply:
My intent was to keep the original source code as intact as possible, so I decided the disable the rule for the line below instead of changing it. I have no problem in fixing it though, if you prefer :)

Reply:
No, what you have is ok. Was just wondering whether there was another reason you disabled it.